src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -46,6 +46,12 @@ internal static class LogMessages
         public const string IDX10103 = "IDX10103: RoleClaimType cannot be null or whitespace.";
         public const string IDX10104 = "IDX10104: TokenLifetimeInMinutes must be greater than zero. value: '{0}'";
         public const string IDX10105 = "IDX10105: ClaimValue that is a collection of collections is not supported. Such ClaimValue is found for ClaimType : '{0}'";
+        public const string IDX10106 = "IDX10106: Using this property to alter the default value and disable expiry requirements is not recommended. https://aka.ms/WilsonTokenValidation";
```

Comment:
nit: we have aka.ms/wilson/ path reserved. Perhaps it's more consistent to use aka.ms/wilson/tokenvalidation in this case.